vendor/github.com/sam-falvo/mbox/mbox.go

```diff
@@ -197,7 +197,13 @@ func (m *MboxStream) nextLine() error {
 	if err != nil {
 		return err
 	}
-	m.prefetch = m.prefetch[0:len(slice)]
+
+	// If slice is longer than the original prefetch, resize.
+	if len(slice) < 1000 {
```

Comment:
I think this should be `<=`. Also, although I think this works, do we want to compare to `cap(m.prefetch)`? If there are multiple long lines, we may have already extended, and don't need to extend each time.

Reply:
Yeah good idea, haven't thought about that function in a while